crates/solidity/inputs/language/src/definition.rs

```diff
@@ -4074,6 +4075,118 @@ codegen_language_macros::compile!(Language(
+                                EnumVariant(reference = YulLog3Keyword),
+                                EnumVariant(reference = YulLog4Keyword),
+                                EnumVariant(reference = YulLtKeyword),
+                                EnumVariant(reference = YulMloadKeyword),
+                                EnumVariant(reference = YulModKeyword),
+                                EnumVariant(reference = YulMsizeKeyword),
+                                EnumVariant(reference = YulMstoreKeyword),
+                                EnumVariant(reference = YulMstore8Keyword),
+                                EnumVariant(reference = YulMulKeyword),
+                                EnumVariant(reference = YulMulmodKeyword),
+                                EnumVariant(reference = YulNotKeyword),
+                                EnumVariant(reference = YulNumberKeyword),
+                                EnumVariant(reference = YulOrKeyword),
+                                EnumVariant(reference = YulOriginKeyword),
+                                EnumVariant(reference = YulPopKeyword),
+                                EnumVariant(reference = YulReturnKeyword),
+                                EnumVariant(reference = YulRevertKeyword),
+                                EnumVariant(reference = YulSdivKeyword),
+                                EnumVariant(reference = YulSelfdestructKeyword),
+                                EnumVariant(reference = YulSgtKeyword),
+                                EnumVariant(reference = YulSignextendKeyword),
+                                EnumVariant(reference = YulSloadKeyword),
+                                EnumVariant(reference = YulSltKeyword),
+                                EnumVariant(reference = YulSmodKeyword),
+                                EnumVariant(reference = YulSstoreKeyword),
+                                EnumVariant(reference = YulStopKeyword),
+                                EnumVariant(reference = YulSubKeyword),
+                                EnumVariant(reference = YulTimestampKeyword),
+                                EnumVariant(reference = YulXorKeyword),
+                                // Renamed keccak256 in 0.4.12, sha3 alias removed in 0.5.0
+                                EnumVariant(reference = YulSha3Keyword, enabled = Till("0.5.0")),
+                                // Introduced as alias to selfdestruct in 0.2.0, removed in 0.5.0
+                                EnumVariant(reference = YulSuicideKeyword, enabled = Till("0.5.0")),
+                                EnumVariant(
+                                    reference = YulKeccak256Keyword,
+                                    enabled = From("0.4.12")
+                                ),
+                                // Introduced in byzantium
+                                EnumVariant(
+                                    reference = YulReturndatacopyKeyword,
```

Comment:
nit: I suggest capitalizing words like this (and a few others) for readability:

```suggestion
                                    reference = YulReturnDataCopyKeyword,
```